src/crypto/algorithms/megolm.ts

```diff
@@ -1795,12 +1795,16 @@ class MegolmDecryption extends DecryptionAlgorithm {
      * @private
      * @param {String} senderKey
      * @param {String} sessionId
-     * @param {Boolean} keyTrusted
+     * @param {Boolean} forceRedecryptIfUntrusted
```

Comment:
I'm not sure this is much better, filling the docstring would be more appreciated

Reply:
The rename was suggested by @BillCarsonFr on the private gitlab repo; adding the docstring is a good idea in any case :+1: